hazelcast/src/main/java/com/hazelcast/config/NativeMemoryConfig.java

```diff
@@ -16,13 +16,18 @@
 
 package com.hazelcast.config;
 
+import com.hazelcast.core.HazelcastException;
```

Comment:
I just realized we're missing a read-only version of `NativeMemoryConfig` but other top-level configs have this issue as well.

Reply:
Yeah, as this is a general concern, I created https://github.com/hazelcast/hazelcast/issues/17372 as discussed in private.